src/main/resources/application.yaml

```diff
@@ -124,6 +124,7 @@ azure:
 
 features:
   federated-s-s-o: ${federated.sso:true}
+  step-up-authentication: ${step.up.authentication:true}
```

Comment:
Needs to be `true` by default for functional tests. It can then be controlled by a flux entry.